<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>771</x>
      <y>526</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>CryoTheraPy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="group_jobs">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>561</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Jobs</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>151</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="lay_job1_import_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path to Files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_imp_path"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Path to Meta</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_imp_meta"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pixel Size (A)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Job 1: Import</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line_imp_pix_size"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>qsub scrips</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_imp_qsub"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>151</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="lay_job2_motioncorr">
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>EER Grouping</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nr MPI</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Nr Threads</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_mcorr_eer"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_mcorr_mpi"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line_mcorr_threads"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Job 2: MotionCorr</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>qsub scrips</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_mcorr_qsub"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_changeStarValues">
     <property name="geometry">
      <rect>
       <x>439</x>
       <y>260</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Commit Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_readStar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Paths</string>
    </property>
    <widget class="QPushButton" name="btn_writeStar">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Write Star</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_path_new_project"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Path to new Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_python2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>490</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch to Python 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Scheme">
    <property name="title">
     <string>&amp;Menu Scheme</string>
    </property>
    <addaction name="actionSave_Progress"/>
    <addaction name="actionEdit_Base_Parameters"/>
   </widget>
   <addaction name="menu_Scheme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Progress">
   <property name="text">
    <string>Save Progress</string>
   </property>
  </action>
  <action name="actionEdit_Base_Parameters">
   <property name="text">
    <string>Edit Base Parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
